File: src/ERP/acme.atena.domain/DTO/Inventory/EstoqueProduto.cs
using acme.atena.domain.DTO.Product;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace acme.atena.domain.DTO.Inventory
{
    public class EstoqueProduto : AbstractEntity
    {
        public Guid ProdutoId { get; set; }
        public Guid EstoqueId { get; set; }
        public long QuantidadeMaxima { get; set; }
        public long QuantidadeMinima { get; set; }
        public long QuantidadeProduto { get; set; }

        public virtual Estoque Estoque { get; set; }
        public virtual Produto Produto { get; set; }
    }
}

File: site/coreui-free-angular-admin-template-master/src/app/view-model/response/inventory/estoque-produto.ts
import { Produto } from './../product/produto';
import { Estoque } from './estoque';
import { AbstractEntity } from "../abstract-entity";

export class EstoqueProduto extends AbstractEntity{
  public produtoId: string ;
  public estoqueId: string ;
  public quantidadeMaxima: number;
  public quantidadeMinima:number;
  public quantidadeProduto:number;

  public estoque: Estoque ;
  public produto: Produto ;
}
